skycultures/indian/index.json

```diff
@@ -728,98 +728,99 @@
-			 { "symbol": "17", "native": "अनुराधा",           "pronounce": "Anuradha",    "english": "Following rādhā"},
-			 { "symbol": "18", "native": "ज्येष्ठा",             "pronounce": "Jyeshtha",   "english": "Eldest, most excellent"},
-			 { "symbol": "19", "native": "मूल",              "pronounce": "Mula",       "english": "Root"},
-			 { "symbol": "20", "native": "पूर्व आषाढा",          "pronounce": "Purva Ashadha",  "english": "First Invincible"},
-			 { "symbol": "21", "native": "उत्तर आषाढा",         "pronounce": "Uttara Ashadha", "english": "Later Invincible"},
-			 { "symbol": "22", "native": "श्रवण",             "pronounce": "Shravana",       "english": ""},
-			 { "symbol": "23", "native": "श्रविष्ठा/धनिष्ठा",         "pronounce": "Dhanishta/Shravishthā", "english": "Most Famous/Swiftest"},
-			 { "symbol": "24", "native": "शतभिष/शततारका",      "pronounce": "Shatabhisha Nakshatra", "english": "Requiring a hundred Physicians"},
-			 { "symbol": "25", "native": "पूर्व भाद्रपदा/पूर्व प्रोष्ठपदा",   "pronounce": "Purva Bhadrapada",  "english": "First of the Blessed Feet"},
-			 { "symbol": "26", "native": "उत्तर भाद्रपदा/उत्तर प्रोष्ठपदा", "pronounce": "Uttara Bhādrapadā", "english": "Second of the Blessed Feet"},
-			 { "symbol": "27", "native": "रेवती",            "pronounce": "Revati",            "english": "Prosperous"}, 
-			 { "symbol": "28", "native": "अभिजित",          "pronounce": "Abhijit",           "english": "Invincible"}]},			 
+	"names": [ { "symbol":  "Aś", "native": "अश्विनी", "pronounce": "Aśvinī",    "english": "Physician to the Gods"},
+			 { "symbol":  "Bh", "native": "भरणी", "pronounce": "Bharaṇī",    "english": "Bearer"},
+			 { "symbol":  "Kr̥", "native": "कृत्तिकाः", "pronounce": "Kr̥ttikā",    "english": "Pleiades"},
+			 { "symbol":  "Ro", "native": "रोहिणी", "pronounce": "Rohiṇī",     "english": "Red"},
+			 { "symbol":  "Mr̥", "native": "मृगशिराः", "pronounce": "Mr̥gaśiras", "english": "Deer's Head"},
+			 { "symbol":  "Ār", "native": "आर्द्रा", "pronounce": "Ārdrā",      "english": "Wet"},
+			 { "symbol":  "Pv", "native": "पुनर्वसू", "pronounce": "Punarvasū",  "english": "Two Restorers of Goods"},
+			 { "symbol":  "Puṣ", "native": "पुष्यम्", "pronounce": "Puśya",    "english": "Nourisher"},
+			 { "symbol":  "Āś", "native": "आश्लेषा", "pronounce": "Āshleshā",   "english": "Embrace"},
+			 { "symbol": "Ma", "native": "मघाः", "pronounce": "Maghā",     "english": "Bountiful"},
+			 { "symbol": "PPh", "native": "पूर्वफाल्गुनी", "pronounce": "Pūrva phālgunī", "english": "Former Fruitful"},
+			 { "symbol": "UPh", "native": "उत्तरफाल्गुनी", "pronounce": "Uttara phālgunī", "english": "Latter Fruitful"},
+			 { "symbol": "Ha", "native": "हस्तः", "pronounce": "Hasta",     "english": "Hand"},
+			 { "symbol": "Cit", "native": "चित्रा", "pronounce": "Chitrā",    "english": "Bright One"},
+			 { "symbol": "15", "native": "स्वाती", "pronounce": "Svātī",     "english": "Self mover"},
```

Comment:
Once more, I suggest numerical "symbols" here, as these are always visible next to the full names and might make orientation easier at least for non-natives. If not, number 15 should probably also be replaced.